index.html

```diff
@@ -35,7 +35,7 @@
               <p>Tickets are $10 (includes a donut), and 100% of proceeds go to a local non-profit.</p>
               <p class='non-prof-description'>This month's non-profit is <a href='http://portland.chicktech.org/'>ChickTech Portland</a>, a non-profit that provides mentoring and workshops to Portland area girls interested in STEM fields.</p>
 
-              <a class="navigation-link navigation-link-blue" href="./tickets/index.html">Buy Tickets</a>
+              <a class="navigation-link navigation-link-blue" href="./tickets/">Buy Tickets</a>
```

Comment:
hmm not sure why, but without putting 'index.html' to the end of the link I get directed to this weird page: 
![image](https://cloud.githubusercontent.com/assets/8960591/13764382/e5c30bfa-ea0b-11e5-8e5b-7ebd0ead6dd9.png)